src/content-scripts/attachmentFrame.js

```diff
@@ -35,6 +35,9 @@ export default class AttachmentFrame extends ExtractFrame {
       this.pgpElement.classList.add('m-extract-wrapper');
     }
     this.pgpElement.style.display = 'inline-block';
+    // let the element to take as much as it wants if there is a message text, like in `decryptFrame`
+    // but limit it in case it is empty; we then do simillar for the height in `#setFrameDim()`
```

Comment:
- "limit" sounds like we define a max-height, but actually we define a min-height. That means even if empty it will always have height 400px?
- typo: simillar
- '#setFrameDim()' -> as # is used as a prefix for private methods in JS I would not use it as a kind of hash. 'setFrameDim()' should be fine

Reply:
* Fixed
* Fixed
* Sorry, this comes from Javadoc, where it means a method, `JsClass#method`. Fixed

Reply:
- similar is still simillar ;)
- I didn't know about https://jsdoc.app/about-namepaths. If it's a valid syntax you can of course use it.

Reply:
Fixed, sorry